app/experimenter/targeting/constants.py

```diff
@@ -142,6 +142,33 @@ def __post_init__(self):
     application_choice_names=(Application.DESKTOP.name,),
 )
 
+FIRST_RUN_NEW_PROFILE_HAS_PIN_NEED_DEFAULT_WINDOWS_1903 = NimbusTargetingConfig(
+    name=(
+        "First start-up users on Windows 10 1903 (build 18362) or newer, with a "
+        "new profile, needing default w/ pin"
+    ),
+    slug="first_run_new_profile_need_default_has_pin",
+    description=(
+        "First start-up users (e.g. for about:welcome) on Windows 1903+, "
+        "with a new profile, needing default w/ pin"
+    ),
+    targeting=(
+        "{first_run} && {has_pin}".format(
+            first_run=FIRST_RUN_NEW_PROFILE_NEED_DEFAULT_WINDOWS_1903.targeting,
+            has_pin=HAS_PIN,
```

Comment:
Any reason we are using ```.format``` over f-strings?

Reply:
No particular reason, was basing this targeting off of `FIRST_RUN_NEW_PROFILE_NEED_DEFAULT_WINDOWS_1903_PREFER_MOTION`. But I can update it if f-strings is the more common format.

Reply:
Oh yeah, I've thought about that, in general yes fstrings are preferred over .format but in this case, the lookups for the fields being substituted in can be very long and it would make the fstrings much uglier and difficult to read so I think .format is fine in this context.